android/src/main/java/com/opentokreactnative/OTSessionManager.java

```diff
@@ -4,14 +4,16 @@
  * Created by manik on 1/29/18.
  */
 
+import android.os.Build;
 import android.util.Log;
 import android.widget.FrameLayout;
 import android.view.View;
 
 import androidx.annotation.Nullable;
-
+import androidx.annotation.RequiresApi;
```

Comment:
@cnx-carlosvogt cnx-carlosvogt Looks. I'd like to merge this is. Is RequiresApi used? I don't see it referenced in the code.

Reply:
Hey, @jeffswartz . I work with @cnx-carlosvogt and he is on vacation now. Yes, we can proceed with this deletion.

Reply:
Great. Thanks, @cnx-renansoares. I am merging this in and will create a new version of the library with fix. And thanks again @cnx-carlosvogt for fixing this.